src/index.html

```diff
@@ -6,13 +6,60 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image-container">
```

Comment:
This div is redundant
```suggestion
```

Reply:
I don't know how to make alignment in a card block without this container

Reply:
you can always ask about it in the chat
<img width="627" alt="image" src="https://github.com/user-attachments/assets/1e9540e3-a871-4bb0-9f3f-e8f04e9865a5">